lib/modules.js

```diff
@@ -91,12 +91,48 @@ export const installBinaryModule = async ({
   console.log(`[${module}] ✓ ${outFile}`)
 }
 
-export async function downloadSourceFiles ({ module, repo, distTag }) {
+/** @typedef {{
+ *   tag_name: string
+ * }?} GitHubRelease */
```

Comment:
If I am reading this type correctly, it says that GitHubRelease is either an object with `tag_name` property, or `undefined`.

Later, you have:

```js
 const body = /** @type {GitHubRelease} */ (await res.json())
 return body.tag_name
```

Under which conditions can `res.json()` return undefined? 

If `body` can be undefined, shouldn't you use `?.` to read its properties?


```js
return body?.tag_name
```

Reply:
You're right, this can't be `undefined`